### PR TITLE
Bossy.treeview

### PR DESCRIPTION
**Pull request by rahulhgaikwad**
Work in Progress. Event handling logic has to be implemented.  #277

### CODE REVIEW

**Comment by rahulhgaikwad** [comment]
Work is in progress.

File: ng2/bossy.treeview/bossy.treeview.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+@Component({
+    selector: 'bossy-treeview',
+    templateUrl: 'bossy-treeview.html',
+    inputs: ['config'],
+    styleUrls: ['_bossy-treeview.css']
+})
+
+export  class BossyTreeview implements OnInit {
+    public config: BossyTreeviewConfig;
+}
```
Comment (tmjam) [inline ng2/bossy.treeview/bossy.treeview.ts]: Missing logic for handling events.

**Comment by tmjam** [comment]
Missing event handling logic. Waiting for it to be completed.

**Comment by glawson** [comment]
@rahulhgaikwad , @tmjam asked about event handling logic - is it done?

**Comment by rahulhgaikwad** [comment]
@glawson Not yet.

**Pull request closed without merging**